File: 程式語言 V2答案檔/Python/106.py
import math
from decimal import Decimal, ROUND_HALF_UP

x1 = int(input())
y1 = int(input())
x2 = int(input())
y2 = int(input())

distance_tmp = math.sqrt((x1 - x2) ** 2 + (y1 - y2) ** 2)
distance = Decimal(str(distance_tmp)).quantize(Decimal('0.00'), rounding = ROUND_HALF_UP)
print(distance)

File: 程式語言 V2答案檔/Go/106.go
package main
import (
	"fmt"
	"math"
)

func main(){
	var a, b, c, d float64
	fmt.Scanln(&a)
	fmt.Scanln(&b)
	fmt.Scanln(&c)
	fmt.Scanln(&d)
	fmt.Printf("%.2f", math.Sqrt((a-c)*(a-c)+(b-d)*(b-d)))
}